server/chat-commands/room-settings.ts

```diff
@@ -915,6 +914,7 @@ export const commands: ChatCommands = {
 		if (room.persist && isGroupchat) return this.errorReply(`This isn't a groupchat.`);
 		if (!room.persist && !isGroupchat) return this.errorReply(`Use /renamegroupchat instead.`);
 		if (isGroupchat) {
+			if (!this.can('modchat', null, room)) return false;
```

Comment:
I mean, this is a very elegant way of making sure both Hosts and global Drivers can rename rooms, but I'm not sure I like this.

```suggestion
			if (!user.can('mute) && !this.can('editroom', null, room)) return false;
```